test_cromwell.sh

```diff
@@ -120,8 +120,12 @@ sbt test:compile
 CP=$(sbt "export test:dependency-classpath" --error)
 
 if [[ -n ${EXCLUDE_TAG} ]]; then
```

Comment:
This also triggered a shellcheck warning since it's now an array: https://github.com/koalaman/shellcheck/wiki/SC2128